float64 x
float64 y
float64 z
float64 wx
float64 wy
float64 wz

---
bool success
